src/lib/routes/admin-api/client-metrics.ts

```diff
@@ -18,39 +32,79 @@ class ClientMetricsController extends Controller {
         config: IUnleashConfig,
         {
             clientMetricsServiceV2,
-        }: Pick<IUnleashServices, 'clientMetricsServiceV2'>,
+            openApiService,
+        }: Pick<IUnleashServices, 'clientMetricsServiceV2' | 'openApiService'>,
     ) {
         super(config);
         this.logger = config.getLogger('/admin-api/client-metrics.ts');
 
         this.metrics = clientMetricsServiceV2;
+        this.openApiService = openApiService;
+
+        this.route({
+            method: 'get',
+            path: '/features/:name/raw',
+            handler: this.getRawToggleMetrics,
```

Comment:
Would rename this handler and operationId to `getRawFeatureMetrics` while were here, to match the schema.